--- hdf5-1.10.5.orig/testpar/Makefile.in	2019-04-28 10:37:40.229340824 +0200
+++ hdf5-1.10.5/testpar/Makefile.in	2019-04-28 10:40:54.703202501 +0200
@@ -806,7 +806,7 @@
 
 # Test programs.  These are our main targets.
 #
-TEST_PROG_PARA = t_mpi t_bigio testphdf5 t_cache t_cache_image t_pread t_pshutdown t_prestart t_init_term t_shapesame t_filters_parallel
+TEST_PROG_PARA = t_mpi t_bigio t_pread t_pshutdown t_prestart t_init_term
 testphdf5_SOURCES = testphdf5.c t_dset.c t_file.c t_file_image.c t_mdset.c \
 	t_ph5basic.c t_coll_chunk.c t_span_tree.c t_chunk_alloc.c t_filter_read.c \
 	t_prop.c t_coll_md_read.c
