--- hdf5-1.10.5.orig/test/Makefile.in	2019-04-28 10:37:40.222674147 +0200
+++ hdf5-1.10.5/test/Makefile.in	2019-04-28 10:38:23.012748393 +0200
@@ -1436,8 +1436,8 @@
 # As an exception, long-running tests should occur earlier in the list.
 # This gives them more time to run when tests are executing in parallel.
 TEST_PROG = testhdf5 \
-           cache cache_api cache_image cache_tagging lheap ohdr stab gheap \
-           evict_on_close farray earray btree2 fheap \
+           cache_api cache_tagging lheap ohdr stab gheap \
+           evict_on_close farray earray btree2 \
            pool accum hyperslab istore bittests dt_arith page_buffer \
            dtypes dsets chunk_info cmpd_dset filter_fail extend direct_chunk \
            external efc objcopy links unlink twriteorder big mtime fillval mount \
